style.css

```diff
@@ -0,0 +1,205 @@
+.app-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 540px auto 90px;
+  grid-row-gap: 2px;
+  grid-template-areas:
+    "main main main main"
+    "iphone5c iphone5s mac supplier"
+    "footer footer footer footer";
+}
+
+/* Main section */
```

Comment:
- [ ] Required - Remove inline comments in your code if you want to refer to a previous version of your code please use git history. Therefore remove these comments and all other similar comments.